src/cbmc/bmc.cpp

```diff
@@ -566,12 +567,12 @@ int bmct::do_language_agnostic_bmc(
 
     while(!worklist->empty())
     {
-      cbmc_solverst solvers(
+      solver_factoryt solvers(
         opts,
         symbol_table,
         ui,
         ui.get_ui() == ui_message_handlert::uit::XML_UI);
-      std::unique_ptr<cbmc_solverst::solvert> cbmc_solver;
+      std::unique_ptr<solver_factoryt::solvert> cbmc_solver;
       cbmc_solver = solvers.get_solver();
```

Comment:
:pick: assignment could be on same line as declaration

Reply:
Done